src/utils/plot-templates/dark.js

```diff
@@ -0,0 +1,229 @@
+//Plotly templates are defined to override user-specified styles with Kedro-viz styles
```

Comment:
I think best to use a block style comment here and the other file (as just a general style convention for file level docs)